scripts/eks-helm-test.sh

```diff
@@ -0,0 +1,196 @@
+                                      Default: ack-system
+  SKIP_PYTHON_TESTS:                  Whether to skip python tests and run bash tests instead for
+                                      the service controller (<true|false>)
+                                      Default: false
+  RUN_PYTEST_LOCALLY:                 If python tests exist, whether to run them locally instead of
+                                      inside Docker (<true|false>)
+                                      Default: false
+"
+
+if [ $# -ne 1 ]; then
+    echo "AWS_SERVICE is not defined. Script accepts one parameter, <AWS_SERVICE>" 1>&2
+    echo "${USAGE}"
+    exit 1
+fi
+
+AWS_SERVICE=$(echo "$1" | tr '[:upper:]' '[:lower:]')
+
+if [ -z "$EKS_CLUSTER_NAME" ]; then
+    echo "No Amazon EKS cluster name specified." 1>&2
+    echo "${USAGE}"
+    exit  1
+fi
+
+if [ -z "$EKS_CLUSTER_NAME" ]; then
+    echo "No Amazon EKS cluster name specified." 1>&2
+    echo "${USAGE}"
+    exit  1
+fi
+
+if [ -z "$ACK_CONTROLLER_RELEASE_VERSION" ]; then
+    echo "No release version specified for ACK $AWS_SERVICE controller." 1>&2
+    echo "${USAGE}"
+    exit  1
+fi
+
+if [ -z "$IRSA_ROLE_ARN" ]; then
+    echo "No IRSA Role ARN specified to setup service account for ACK $AWS_SERVICE controller." 1>&2
+    echo "${USAGE}"
+    exit  1
+fi
+
+if [ -z "$ACK_K8S_SERVICE_ACCOUNT_NAME" ]; then
+    ACK_K8S_SERVICE_ACCOUNT_NAME="ack-${AWS_SERVICE}-controller"
+fi
+
+if [ -z "$TMP_DIR" ]; then
+    TEST_ID=$(uuidgen | cut -d'-' -f1 | tr '[:upper:]' '[:lower:]')
+    TMP_DIR=$ROOT_DIR/build/tmp-eks-test-$TEST_ID
+fi
+
+echo "To run the tests, using temporary directory: $TMP_DIR"
+mkdir -p "$TMP_DIR"
+
+KUBECONFIG_PATH="${TMP_DIR}/kubeconfig"
+daws eks update-kubeconfig --name "$EKS_CLUSTER_NAME" --kubeconfig "$KUBECONFIG_PATH"
+if [[ ! -f "$KUBECONFIG_PATH" ]]; then
+    echo "Failed to setup kubeconfig for Amazon EKS cluster: $EKS_CLUSTER_NAME" 1>&2
+    echo "${USAGE}"
+    exit 1
+fi
+
+KUBECONFIG="$KUBECONFIG_PATH"
+export KUBECONFIG
+
+echo "======================================================================================================"
+echo "To poke around your test cluster manually:"
+echo "export KUBECONFIG=$KUBECONFIG_PATH"
+echo "kubectl get pods -A"
+echo "======================================================================================================"
+
+# install helm chart
+ACK_K8S_RELEASE_NAME="ack-${AWS_SERVICE}-controller"
+REMOTE_HELM_CHARTS="$HELM_REGISTRY/$HELM_REPO:${AWS_SERVICE}-$ACK_CONTROLLER_RELEASE_VERSION"
+
+HELM_EXPERIMENTAL_OCI=1
+export HELM_EXPERIMENTAL_OCI
+
+echo "Installing helm charts: $REMOTE_HELM_CHARTS on Amazon EKS cluster: $EKS_CLUSTER_NAME"
+echo "Helm chart install dryrun output location: ${TMP_DIR}/helm_install_dryrun_output"
+
+LOCAL_HELM_CHARTS="${TMP_DIR}/charts"
+
+helm chart pull "$REMOTE_HELM_CHARTS"
+helm chart export "$REMOTE_HELM_CHARTS" --destination "$LOCAL_HELM_CHARTS"
```

Comment:
This will only test the released versions of the helm charts? Not the one in the local repository?